test/dawg.test.js

```diff
@@ -101,12 +115,19 @@ test('Read-write DAWG test', function(t) {
     t.assert(!dawg.lookup(""), "dawg does not contain the empty string as a term");
     t.assert(dawg.lookupPrefix(""), "dawg does contain the empty string as a prefix");
 
+    dawg.finish();
+
+    // I don't care about the values these functions return at all... they exist
+    // only for debugging and are not exercised ever in typical usage.
+    // but, run them anyway, and ensure that, I dunno, they don't crash? so that
+    // the coverage tool shuts up
```

Comment:
You can also exclude things from coverage in the C++ code that don't matter by wrapping in `/* LCOV_EXCL_START */` and `/* LCOV_EXCL_STOP */` like https://github.com/mapnik/node-mapnik/blob/8501d014c78d9bb1029c57590b13b0fbe405d785/src/mapnik_geometry.cpp#L145-L148

Reply:
Oh cool. I might leave these particular ones in because why not, but there are a couple of other error checks that are in the "I don't think it should ever be possible to reach this point, but just in case..." category that I might like to skip since as far as I know they should otherwise be impossible to test.